src/app-layout/__integ__/app-layout-drawers.test.ts

```diff
@@ -83,43 +100,67 @@ for (const visualRefresh of [true, false]) {
 
     test(
       `should not allow resize drawer beyond min and max limits`,
-      setupTest({ visualRefresh: false }, async page => {
-        await page.openPanel();
-        const { width } = await page.getWindowSize();
-        await page.dragResizerTo({ x: width, y: 0 });
-        expect((await page.getDrawerSize()).width).toEqual(280);
-
-        await page.dragResizerTo({ x: 0, y: 0 });
-        expect((await page.getDrawerSize()).width).toEqual(520);
-      })
-    );
-
-    // in VR, there is a border that adds 2px to the width.
-    test(
-      `should not allow resize drawer beyond min and max limits`,
-      setupTest({ visualRefresh: true }, async page => {
-        await page.openPanel();
+      setupTest({}, async page => {
+        await page.openFirstDrawer();
         const { width } = await page.getWindowSize();
         await page.dragResizerTo({ x: width, y: 0 });
-        expect((await page.getDrawerSize()).width).toEqual(292);
+        // there are different layouts between these two designs
+        await expect(page.getActiveDrawerWidth()).resolves.toEqual(visualRefresh ? 292 : 280);
```

Comment:
Merged two nearly-identical test into a single one, with variable assertion for each design